src/Microsoft.ML.StandardLearners/StandardLearnersCatalog.cs

```diff
@@ -19,15 +19,22 @@ namespace Microsoft.ML
     public static class StandardLearnersCatalog
     {
         /// <summary>
-        ///  Predict a target using logistic regression trained with the <see cref="SgdBinaryTrainer"/> trainer.
+        /// Predict a target using a linear classification model trained with <see cref="SgdBinaryTrainer"/>.
         /// </summary>
-        /// <param name="catalog">The binary classificaiton catalog trainer object.</param>
-        /// <param name="labelColumnName">The name of the label column.</param>
-        /// <param name="featureColumnName">The name of the feature column.</param>
-        /// <param name="weightColumnName">The name for the example weight column.</param>
-        /// <param name="maxIterations">The maximum number of iterations; set to 1 to simulate online learning.</param>
-        /// <param name="initLearningRate">The initial learning rate used by SGD.</param>
-        /// <param name="l2Weight">The L2 regularization constant.</param>
+        /// <param name="catalog">The binary classification catalog trainer object.</param>
+        /// <param name="labelColumnName">The name of the label column, or dependent variable.</param>
+        /// <param name="featureColumnName">The features, or independent variables.</param>
+        /// <param name="weightColumnName">The name of the example weight column.</param>
```

Comment:
>weightColumnName [](start = 25, length = 16)

[https://github.com/dotnet/machinelearning/pull/2665](https://github.com/dotnet/machinelearning/pull/2665)
`/// <param name="exampleWeightColumnName">The name of the example weight column (optional).</param>`

Reply:
thanks for the heads up. I'll leave this for when resolving merge conflicts.

---
In reply to: [259444080](https://github.com/dotnet/machinelearning/pull/2688#discussion_r259444080) [](ancestors = 259444080)